apps/src/componentLibrary/common/contexts/DropdownContext.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {createContext, useContext, useState, ReactNode} from 'react';
+
+const DropdownContext = createContext({
+  activeDropdownName: '',
+  setActiveDropdownName: (name: string) => {
+    console.log(name);
+  },
+});
+
+export const useDropdownContext = () => useContext(DropdownContext);
+
+export const DropdownProvider: React.FC<{children: ReactNode}> = ({
+  children,
+}) => {
+  const [activeDropdownName, setActiveDropdownName] = useState('');
+
+  return (
+    <DropdownContext.Provider
```

Comment:
Not super familiar with context in React...help me understand what this does?

Reply:
This creates 'redux' like storage/context of data shared across components that are wrapped in DropdownProvider like this: 
```
<DropdownProvider>
    <CheckboxDropdown {...props} />
  </DropdownProvider>
```

and then accessed inside DropdownProvider child compoent like this:
`  const {activeDropdownName, setActiveDropdownName} = useDropdownContext();`

This serves a purpose of creating one state for components using `DropdownProvider` at once. Also allows us to share this context with IconsDropdown (to be implemented). (not applicable to `SimpleDropdown` cause native select elements)

And the initial reason behind all of this is that we want only one dropdown to be open at once.

Reply:
Got it, thanks! Is there a reason we're especially conscious not to have multiple of these open at once, especially considering `SimpleDropdown` won't have the same behavior?